test/martha/martha_v3.test.js

```diff
@@ -160,25 +417,25 @@ test.serial.afterEach(() => {
 });
 
 test.serial('martha_v3 resolves a valid DOS-style url', async (t) => {
-    getJsonFromApiStub.onFirstCall().resolves(dataObjectServiceObject);
+    getJsonFromApiStub.onFirstCall().resolves(jadeDosResponse);
     const response = mockResponse();
     await marthaV3(mockRequest({ body: { 'url': 'dos://abc/123' } }), response);
     const result = response.send.lastCall.args[0];
-    t.deepEqual(Object.assign({}, result), fullExpectedResult(googleSAKeyObject));
+    t.deepEqual(Object.assign({}, result), jadeDosMarthaResult(googleSAKeyObject));
     t.is(response.statusCode, 200);
 });
 
 test.serial('martha_v3 resolves a valid DRS-style url', async (t) => {
-    getJsonFromApiStub.onFirstCall().resolves(dataObjectServiceObject);
+    getJsonFromApiStub.onFirstCall().resolves(jadeDosResponse);
```

Comment:
Is this really correct that a DRS url resolves to DOS?  If so, maybe add a comment to this test explaining why this odd result is correct?